packages/apps/reputation-oracle/server/src/common/enums/user.ts

```diff
@@ -0,0 +1,10 @@
+export enum UserStatus {
+  ACTIVE = 'ACTIVE',
+  INACTIVE = 'INACTIVE',
+  PENDING = 'PENDING',
+}
+
+export enum UserType {
```

Comment:
For this case userTypes would be Operator, Exchange Oracle, Recording Oracle and Worker